manifest.yml

```diff
@@ -1,28 +1,29 @@
 artifacts:
- - name: DwsimConnectorInstaller-{version}.msi
-   path: DwsimConnectorInstaller-{version}.msi
-   platform: windows
- - name: DwsimConnector-{version}.pdf
-   path: Documentation.pdf
```

Comment:
why removing pdf?

Reply:
I noticed we only had it for some of the connectors, so it was a consistency update. I never seen where this PDF was made available anyway. In any case, once we have dedicated docs for all the connectors on the public docs, should we maintain the PDF as well?

Reply:
it's on the extractors page.
not sure if we have to provide it though, only capability that it gives is that it maintains the history of docs, something that we don't have on the main docs portal. Not sure it's worth keeping for just that single reason.

what was the initial idea behind it?

![Screenshot 2024-11-26 at 13 35 31](https://github.com/user-attachments/assets/506863d4-5f1f-4dfe-b14f-e24f7a2a7706)

Reply:
Humm, I see. I think this was created as a placeholder until we had official public docs, so it should be safe to remove it now.